substrate/frame/staking/src/tests.rs

```diff
@@ -786,7 +790,11 @@ fn double_staking_should_fail() {
 	});
 }
 
+// note: this test will panic due to manual changes in the `Ledger` storage that will trigger the
+// `try_state` checks. The check tat panics runs at the end of the test (`post_check`), thus the
```

Comment:
In some pallets we allow something like `ExtBuilder::default().try_state(false).execute_with(||..)`